frontend/src/stories/crossingListHeader.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import CrossingListHeader from '../Dashboard/CrossingListPage/CrossingListHeader/CrossingListHeader';
+
+
+storiesOf('Crossing List Header', module)
```

Comment:
Can we have a discussion/walkthrough about how storybook works with storyshots (and I can update the readme after). And in the IRL have a talk about testing philosophy.